packages/loader/container-definitions/src/loader.ts

```diff
@@ -241,6 +241,16 @@ export type ILoaderOptions = {
      * Defaults to false.
      */
     provideScopeLoader?: boolean;
+
```

Comment:
It would be great to add description here that it's "AND" between these too options. It's sort of implied by wording (min), but not might be hard to pick it up

Reply:
Added comment.